.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+# Any ignore directives should be uncommented in downstream projects to disable
+# Dependabot updates for the given dependency. Downstream projects will get
+# these updates when the pull request(s) in the appropriate skeleton are merged
+# and Lineage processes these changes.
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    # ignore:
+    #   - dependency-name: actions/cache
+    #   - dependency-name: actions/checkout
+    #   - dependency-name: actions/setup-python
```

Comment:
Should these lines be uncommented?

Reply:
Ack, thanks for catching that. Resolved in [437668f](https://github.com/cisagov/ncats-webui/pull/11/commits/437668f973d5f8c5074e9f2c50a90bba14780cdc).